static/app/views/performance/newTraceDetails/traceDrawer/tabs/traceVitals.tsx

```diff
@@ -45,7 +46,7 @@ export function TraceVitals(props: TraceVitalsProps) {
                 </Tooltip>
                 <div>
                   <div>{t('transaction')}</div>
-                  <TraceDrawerComponents.TitleOp text={op} />
+                  <TraceDrawerComponents.TitleOp text={op + ' - ' + transaction} />
```

Comment:
what happens if there's no transaction?

Reply:
There always will be, the typing is just a bit screwed

Reply:
oh, so `node.value.transaction` will always have a value?

Reply:
I pushed a ternary just to be safe